api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -57,6 +57,21 @@ message SubstitutionFormatString {
     //  }
     //
     google.protobuf.Struct json_format = 2 [(validate.rules).message = {required: true}];
+
+    // Specify a format with command operators to form a html string.
+    // Its details is described in :ref:`format string<config_access_log_format_strings>`.
+    //
+    // .. code-block::
+    //
+    //   html_format: <p>%LOCAL_REPLY_BODY%:%RESPONSE_CODE%:path=$REQ(:path)%</p>
```

Comment:
IIUC, the only difference is that you are setting content type? Would it be preferable to be able to just specify the content-type?

Reply:
@htuch - do you mean adding just `content-type` to `ResponseMapper` as is suggested here -> https://github.com/envoyproxy/envoy/pull/13019#issuecomment-689456789 ?

Reply:
Yeah. I'm thinking there's probably a lot of different variations on content type that might be asked for over time, so adding a special purpose field here is limiting. It makes sense to distinguish text/JSON, since they are structurally different.